aidies-project/client/src/components/CampaignDetail.js

```diff
@@ -91,13 +90,13 @@ const CampaignDetail = ({ needLogin}) => {
         setAnchorEl(null);
     };
     const handleEdit = () => {
-        
-        setAnchorEl(null);
+        history.push(`/campaign/${id}/edit`);
```

Comment:
You should use a `<Link>` instead of an `onClick` handler for the `MenuItem` that uses `handleEdit`!